cmd/notary-signer/protecting_keystore_passphrase.md

```diff
@@ -0,0 +1,90 @@
+Proposal: Protecting keystore passphrase 
+==========================================
+
+
+TL;DR Passphrases used to wrap private keys and store in the keystore DB, also need to be protected.
+
+----------
+
+Table of Contents
+----------
+[TOC]
+
+
+
+Background
+-------------
+
+Currently in Nortary Signer, the signing keys are maintained in memory or are encrypted using AES wrapping with a password and the cipher text is persisted in a MySQL database. 
```

Comment:
Just for the record, the in memory implementation should never be used outside of testing and development.